search.go

```diff
@@ -103,7 +103,7 @@ func (api *Client) _search(ctx context.Context, path, query string, params Searc
 	if params.SortDirection != DEFAULT_SEARCH_SORT_DIR {
 		values.Add("sort_dir", params.SortDirection)
 	}
-	if params.Highlight != DEFAULT_SEARCH_HIGHLIGHT {
+	if params.Highlight {
```

Comment:
`DEFAULT_SEARCH_HIGHLIGHT` is `false`, so this is saying `if params.Highlight != false`, which is the same as `if params.Highlight`.